src/api/auth.js

```diff
@@ -2,11 +2,28 @@ export const getUserData = async (key, pin) => {
   const response = await fetch(`https://randomuser.me/api/`)
   const data = await response.json()
   const mockedUser = data.results[0]
+  //Converting Timestamp values coming from backend into usable input format.
+
+  //Mocked Time stamp in milliseconds
+  const mockedDateTimestamp = new Date().getTime()
+  // initialize new Date object
+  var date_ob = new Date(mockedDateTimestamp)
+  // year as 4 digits (YYYY)
+  var year = date_ob.getFullYear()
+  // month as 2 digits (MM)
+  var month = ("0" + (date_ob.getMonth() + 1)).slice(-2)
+  // date as 2 digits (DD)
+  var date = ("0" + date_ob.getDate()).slice(-2)
+
+  const dateString = `${year}-${month}-${date}`
```

Comment:
I prefer not to use the dayjs library because it seems to work with this code. but if you insist i am ok with that.

But my question is, why is the transformation supposed to be inside the component?